src/content/docs/iast/troubleshooting/unable-to-find-known-vulnerabilities.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Unable to find known vulnerabilities 
+tags:
+    - IAST
+    - New Relic IAST
+    - Troubleshooting
+metaDescription: Unable to find known vulnerabilities 
+freshnessValidatedDate: 2024-07-16
+---
+
+## Problem
+
+I tested an application with known vulnerabilities, but IAST couldn't detect these.
+
+
+## Solution
+<Callout variant="important">
+IAST starts testing when it detects some traffic, so check if your application works out or has any traffic directed at it. Perform the app's UI or API endpoints. 
```

Comment:
I don't think this should be a callout. It seems like a basic part of the troubleshooting solution. Also, can we give some suggestion on where they should look for traffic in the UI?